Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -384,6 +396,35 @@ public static bool HasContents(MicrowaveComponent component)
             return component.Storage.ContainedEntities.Any();
         }
 
+        /// <summary>
+        /// handles the attempted cooking of unsafe objects
+        /// </summary>
+        /// <remarks>
+        /// returns true if the microwave can continue to cook, false if it can't.
+        /// </remarks>
+        private bool HandleUnsafeItems(EntityUid uid, MicrowaveComponent component)
+        {
+            if (!component.IsSparking)
+            {
+                return true;
+            }
+
+            if (component.IsSparking)
+            {
+                if (_random.Prob(.1f))
+                {
+                    _explosion.TriggerExplosive(uid);
+                    return false;  // microwave is fucked, stop the cooking.
+                }
+                if (_random.Prob(.75f))
```

Comment:
Is there a particular reason to otherwise not always spark?

Reply:
I feel like it's more realistic that way.

Reply:
Agreed, that's fine.